languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -59925,6 +59925,64 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Typografie -->
     <!-- ====================================================================== -->
     <category id="TYPOGRAPHY" name="Typografie">
+        <rulegroup id="SPACE_AFTER_BEFORE_SLASH" name="Leerzeichen vor und nach Schrägstrich" default="temp_off">
+            <rule>
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="yes">/</token>
+                    <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                </antipattern>
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="no">/</token>
+                    <token regexp="yes" spacebefore="no">.*[\wüöäß].*</token>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token regexp="yes">.*[\wüöäß].*</token>
+                        <token spacebefore="no">/</token>
+                        <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                    </marker>
+                </pattern>
+                <message>Bei einem Schrägstrich sind Leerzeichen in der Regel nur zulässig, wenn sie sowohl davor als auch dahinter stehen. Alternativ können sie auch ganz weggelassen werden.</message>
+                <suggestion>/</suggestion>
+                <suggestion> / </suggestion>
```

Comment:
Wenn es eine Suggestion gibt (was es sollte), dann muss in der correction auch was stehen 🤓